Sources/DistributedActors/ClusterSystemSettings.swift

```diff
@@ -449,3 +446,32 @@ public struct ServiceDiscoverySettings {
         case dynamic(AnyServiceDiscovery)
     }
 }
+
+// ==== ----------------------------------------------------------------------------------------------------------------
+// MARK: Remote Call Settings
+
+extension ClusterSystemSettings {
+    public struct RemoteCallSettings {
+        public static var `default`: RemoteCallSettings {
+            .init()
+        }
+
+        /// If no other timeout is specified, this timeout is applied to every distributed call.
+        ///
+        /// Set to `.effectivelyInfinite` to avoid setting a timeout, although this is not recommended.
+        public var defaultTimeout: Duration = .seconds(5)
+
+        public var codableErrorAllowance: CodableErrorAllowance = .all
+
+        public enum CodableErrorAllowance {
+            /// All ``Codable`` errors will be converted to ``GenericRemoteCallError``.
+            case none
+
+            /// All ``Codable`` errors will be returned as-is.
+            case all
+
+            /// Only the indicated ``Codable`` errors are allowed. Others are converted to ``GenericRemoteCallError``.
+            case custom([(Error & Codable).Type])
+        }
+    }
+}
```

Comment:
Like the namespace!

looks nice, we'll need to do the annoying enum dance for it sadly ho :)

Reply:
I will do it in this PR

Reply:
amended https://github.com/apple/swift-distributed-actors/pull/1020/commits/046cd443dfd0542aa04117d368e480afe6a49c3b

Reply:
Nice, thanks!